Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/view/child_list/ChildListActivity.java

```diff
@@ -0,0 +1,84 @@
+package pg.autyzm.friendly_plans.view.child_list;
+
+import android.databinding.DataBindingUtil;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import database.repository.ChildRepository;
+import javax.inject.Inject;
+import pg.autyzm.friendly_plans.App;
+import pg.autyzm.friendly_plans.R;
+import pg.autyzm.friendly_plans.databinding.ActivityChildListBinding;
+import pg.autyzm.friendly_plans.notifications.ToastUserNotifier;
+
+public class ChildListActivity extends AppCompatActivity implements ChildListActivityEvents {
+
+    @Inject
+    ChildRepository childRepository;
+
+    @Inject
+    ToastUserNotifier toastUserNotifier;
+
+    ChildListData childData;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        ((App) getApplication()).getAppComponent().inject(this);
+        setContentView(R.layout.activity_child_list);
+
+        ActivityChildListBinding binding = DataBindingUtil
+                .setContentView(this, R.layout.activity_child_list);
+        binding.setChildListEvents(this);
+
+        String initialFirstName = "";
```

Comment:
37,38 variables are unnecessary, it could be only new ChildListData(StringUtils.EMPTY, StringUtils.EMPTY); I think it's worth to avoid additional variables if only it does not affect readability.

Reply:
Sure, you're right :+1:  Thanks ;)